charts/argo-workflows/templates/controller/workflow-controller-deployment.yaml

```diff
@@ -45,7 +45,7 @@ spec:
           command: [ "workflow-controller" ]
           args:
           - "--configmap"
-          - "{{ template "argo-workflows.controller.fullname" . }}-configmap"
+          - {{ template "argo-workflows.controller.config-map.name" . | quote }}
```

Comment:
I wonder we should support for the case, `controller.configMap.create = false` ? 🤔